assets/src/scss/base/_css-variables.scss

```diff
@@ -2,8 +2,8 @@
   --body--font-family: #{$lora};
   --headings--font-family: #{$roboto};
   --headings--font-weight: bold;
-  --body--background-color: #{$white};
-  --body--color: #{$grey-80};
+  --body--background-color: var(--white);
+  --body--color: var(--color-text-body);
```

Comment:
This is the new identity body text color, I don't think that we want to set it here since it should only be applied when the setting is toggled... You could use it [here](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/new-identity/style.scss#L2) though 🙂